### PR TITLE
Fix incorrect referral discount

### PR DESCRIPTION
**Pull request by vipineth**
[x] Fix incorrect referral discount

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205227361086463
  - https://app.asana.com/0/0/1204834612413163
  - https://app.asana.com/0/0/1205276763751836

### CODE REVIEW

File: src/components/Referrals/referralsHelper.js [hunk 1]
```diff
@@ -69,6 +76,26 @@ export const tierDiscountInfo = {
   2: 10,
 };
 
+export const totalRebateInfo = {
+  0: 10,
+  1: 20,
+  2: 25,
+};
```
Comment (divhead) [inline src/components/Referrals/referralsHelper.js:83]: hm, we don't need to hardcode this, beacause we have useTiers hook which requests ReferralStorage for actual totalTebate config

File: src/components/Referrals/referralsHelper.js [hunk 2]
```diff
@@ -10,10 +10,17 @@ import { ARBITRUM, AVALANCHE } from "config/chains";
 import { bigNumberify, formatAmount } from "lib/numbers";
 import { t } from "@lingui/macro";
 import { getRootUrl } from "lib/url";
+import { BASIS_POINTS_DIVISOR } from "config/factors";
 
 export const REFERRAL_CODE_REGEX = /^\w+$/; // only number, string and underscore is allowed
 export const REGEX_VERIFY_BYTES32 = /^0x[0-9a-f]{64}$/;
 
+export function removeTrailingZeros(amount) {
```
Comment (divhead) [inline src/components/Referrals/referralsHelper.js:18]: probably it's better to move this function to lib/numbers

File: src/components/Referrals/referralsHelper.js (hunk 1 above)
Comment (vipineth) [reply src/components/Referrals/referralsHelper.js]: updated!

File: src/components/Referrals/referralsHelper.js (hunk 2 above)
Comment (vipineth) [reply src/components/Referrals/referralsHelper.js]: Updated

**Review by divhead — DISMISSED** [review]

**Review by divhead — APPROVED** [review]

**Pull request merged**